src/app/Http/Middleware/BackpackErrorViews.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Backpack\CRUD\app\Http\Middleware;
+
+use Closure;
+use Illuminate\Support\Str;
+
+class BackpackErrorViews
+{
+    public function handle($request, Closure $next)
+    {
+        if (Str::startsWith($request->path(), config('backpack.base.route_prefix'))) {
+            $themeNamespace = substr(config('backpack.ui.view_namespace'), 0, -2);
+            $themeFallbackNamespace = substr(config('backpack.ui.view_namespace_fallback'), 0, -2);
+            $viewFinderHints = app('view')->getFinder()->getHints();
+
+            $themeErrorPaths = $viewFinderHints[$themeNamespace] ?? [];
+            $themeErrorPaths = $themeNamespace === $themeFallbackNamespace ? $themeErrorPaths :
+                array_merge($viewFinderHints[$themeFallbackNamespace] ?? [], $themeErrorPaths);
+            $uiErrorPaths = [base_path('vendor/backpack/crud/src/resources/views/ui')];
+            $themeErrorPaths = array_merge($themeErrorPaths, $uiErrorPaths);
+
+            app('config')->set('view.paths', array_merge($themeErrorPaths, config('view.paths', [])));
+        }
+
+        return $next($request);
```

Comment:
The fact that all the contents of this method are inside a conditional makes me instantly think "_this can benefit from early returning_". 

```suggestion
        if (!Str::startsWith($request->path(), config('backpack.base.route_prefix'))) {
            return $next($request);
        }
        
        $themeNamespace = substr(config('backpack.ui.view_namespace'), 0, -2);
        $themeFallbackNamespace = substr(config('backpack.ui.view_namespace_fallback'), 0, -2);
        $viewFinderHints = app('view')->getFinder()->getHints();

        $themeErrorPaths = $viewFinderHints[$themeNamespace] ?? [];
        $themeErrorPaths = $themeNamespace === $themeFallbackNamespace ? $themeErrorPaths :
            array_merge($viewFinderHints[$themeFallbackNamespace] ?? [], $themeErrorPaths);
        $uiErrorPaths = [base_path('vendor/backpack/crud/src/resources/views/ui')];
        $themeErrorPaths = array_merge($themeErrorPaths, $uiErrorPaths);

        app('config')->set('view.paths', array_merge($themeErrorPaths, config('view.paths', [])));

        return $next($request);
```

Reply:
But if we include this in the backpack middleware (like I suggested above) we can probably eliminate this conditional entirely.